core-web/libs/sdk/experiments/src/lib/components/react/PageWithExperiments.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactElement, ReactNode } from 'react';
```

Comment:
Are we going to use components? I thought that we were using hooks... well I guess this is just for the placeholder, right?

Reply:
It seems odd to me to add `<DotcmsPage>` and then `<PageWithExperiments>` maybe I'm missing something.

Reply:
Changed to a custom hook